webapp/channels/src/components/searchable_channel_list.tsx

```diff
@@ -588,4 +587,19 @@ export class SearchableChannelList extends React.PureComponent<Props, State> {
     }
 }
 
+const messages = defineMessages({
```

Comment:
Same comment about mixed feelings about having the `defineMessages` here vs having `defineMessage` in every place it is needed.

Nevertheless, if we remove `localizeAndFormatMessage` it may improve things.